src/test/java/io/vertx/ext/dropwizard/MetricsTest.java

```diff
@@ -353,14 +347,14 @@ public void testHttpMetricsOnClose() throws Exception {
     client.close();
     server.close(ar -> {
       assertTrue(ar.succeeded());
-      testComplete();
+      vertx.setTimer(100, tid -> testComplete());
```

Comment:
Thanks Paulo. The code is updated, we just talked about this with Julien on
IRC.

2016-06-09 16:17 GMT+02:00 Paulo Lopes <notifications@github.com>:

> In src/test/java/io/vertx/ext/dropwizard/MetricsTest.java
> <https://github.com/vert-x3/vertx-dropwizard-metrics/pull/32#discussion_r66449064>
> :
>
> > @@ -353,14 +347,14 @@ public void testHttpMetricsOnClose() throws Exception {
> >      client.close();
> >      server.close(ar -> {
> >        assertTrue(ar.succeeded());
> > -      testComplete();
> > +      vertx.setTimer(100, tid -> testComplete());
>
> maybe instead of adding a delay, doing just:
>
> vertx.runOnContext(v -> testComplete());
>
> Is faster and does the same. the runOnContext can also be seen as node.js
> process.nextTick a simple way to execute something on the next event loop
> cycle, this allows the pending operations to terminate.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vert-x3/vertx-dropwizard-metrics/pull/32/files/d8a43519e970bc58e73347cfe2e7e49d41667b24#r66449064>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABbltuhK_6j7nMEWnXVAssExVvvUd_d2ks5qKCBmgaJpZM4Ix-md>
> .
>